deployments/rialto/docker-compose.yml

```diff
@@ -80,6 +80,8 @@ services:
       - ./bridge-config:/config
     environment:
       RUST_LOG: rpc=trace,bridge=trace
+    ports:
+      - "9616:9617"
```

Comment:
Shouldn't it be "9617:9616" instead? According to [documentation](https://docs.docker.com/compose/compose-file/) first number is the port at host machine and it is already occupied by `relay-eth2sub` container (it has "9616:9616" mapping).